app/controllers/application_controller.rb

```diff
@@ -1,9 +1,19 @@
 class ApplicationController < ActionController::Base
   before_action :set_am_host
+  before_action :set_page_title
 
   protect_from_forgery with: :exception
   before_filter :configure_permitted_parameters, if: :devise_controller?
 
+  rescue_from StandardError do |exception|
```

Comment:
Still not convinced about the need of catching exceptions other than `ActiveRecord::RecordNotFound`. While it is clearly good to capture the referrer field for 404 errors (as that will tell us were the broken link is), I do not see the need for capturing the referrer when an other exception occurs. Moreover, if we rescue exceptions this way, we will completely lose a more vital piece of information: the exception itself and its stack trace!